src/components/TopicElements/TopicElements.js

```diff
@@ -156,12 +157,29 @@ function TopicElements({ history, appBaseUrl, staticFilesUrl }) {
 
   const appElements = getComponents(appComponents, elements);
 
+  const getScreenSize = (size) => {
+    let found = false;
+    Object.entries({
+      xs: 576,
+      s: 768,
+      m: 992,
+      l: 1200,
+      xl: Infinity,
+    }).forEach((brkpt) => {
```

Comment:
put this breakpoints in the intial state and give them to the resizehandler. It will avoid differences between the 2